tests/contrib/psycopg/test_psycopg_async.py

```diff
@@ -337,3 +338,26 @@ async def test_cursor_from_connection_shortcut(self):
         rows = await cur.fetchall()
         assert len(rows) == 1, rows
         assert rows[0][0] == "one"
+
+    async def test_cursor_async_connect_execute(self):
```

Comment:
Great regression test for postgres! Can we also add a test case that directly test the TracedAyncCursor class: https://github.com/DataDog/dd-trace-py/blob/ebf3fa99ba5cd58a57fa4c5f1950fb6068754645/tests/contrib/dbapi_async/test_dbapi_async.py#L19.